src/LocalBackend/LocalLibrary.vala

```diff
@@ -922,6 +897,49 @@ public class Noise.LocalLibrary : Library {
         }
     }
 
+    /**
+     * create or load on database and append to or create album and return in-database Media
+     * @param Gee.TreeSet<Media> set of Media
+     * @return Gee.HashMap<int64?, LocalMedia> a map of medias in database
+     */
+    private Gee.HashMap<int64?, LocalMedia> get_local_medias (Gee.TreeSet<Media> medias) {
+        var local_medias = new Gee.HashMap<int64?, LocalMedia> ((x) => { return GLib.int64_hash (x); }, (a, b) => { return GLib.int64_equal (a, b); });
```

Comment:
I'm not 100% sure, but this line could probably be simplified to, while you are touching it:

```vala
var local_medias = new Gee.HashMap<int64?, LocalMedia> (GLib.int64_hash, GLib.int64_equal);
```